doc/topics/development/tests/index.rst

```diff
@@ -415,3 +415,40 @@ question if the path forward is unclear.
     there may be new functionality in the file that is present in the new release branch
     that is untested.It would be wise to see if new functionality could use additional
     testing once the test file has propagated to newer release branches.
+
+
+Test Helpers
+------------
+
+Several Salt-specific helpers are available. A full list is available by inspecting
+functions exported in `salttesting.helpers`.
+
+`@expensiveTest` -- Designates a test which typically requires a relatively costly
```

Comment:
Either remove the `a` on this line, or change `resources` to `resource` in the next line.